docs/src/pages/components/buttons/BusyButtons.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import BusyButton from '@material-ui/core/BusyButton';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Switch from '@material-ui/core/Switch';
+import SaveIcon from '@material-ui/icons/Save';
+import SendIcon from '@material-ui/icons/Send';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& button': {
+      margin: theme.spacing(1),
+    },
+  },
+  switch: {
+    display: 'block',
+  },
+}));
+
+export default function BusyButtons() {
+  const classes = useStyles();
+  const [pending, setPending] = React.useState(false);
+
+  return (
+    <div className={classes.root}>
+      <FormControlLabel
+        control={
+          <Switch
+            checked={pending}
+            onChange={() => setPending(!pending)}
+            name="pending"
+            color="primary"
+          />
+        }
+        className={classes.switch}
+        label="Loading"
+      />
+      <BusyButton variant="outlined" pending={pending} pendingIndicatorPosition="center">
+        Fetch data
```

Comment:
I suggest we inverse the labels to get enough space when loading

<img width="471" alt="Capture d’écran 2020-06-12 à 12 53 52" src="https://user-images.githubusercontent.com/3165635/84495651-ce5fee80-acab-11ea-934e-4552cf49cf37.png">

```suggestion
        Submit
```